continuous_integration/scripts/test_report.py

```diff
@@ -489,7 +489,7 @@ def main(argv: list[str] | None = None) -> None:
         total.groupby([total.file, total.test])
         .filter(lambda g: (g.status == "x").sum() >= args.nfails)
         .reset_index()
-        .assign(test=lambda df: df.file + "." + df.test)
+        .assign(test=lambda df: df.file + "." + df.test)  # type: ignore
```

Comment:
`df.file` is type `Any` and it's not easy to cast this within a lambda so just ignoring it.